app.py

```diff
@@ -44,3 +51,19 @@ def serve_angular(path):
     ])
     return flask.redirect(target)
   return flask.send_file('dist/client/index.html')
+
+@app.route('/list', methods=['GET'])
+def read():
```

Comment:
we should clarify in naming that this is the character list route. It may  make sense to have a file of just character API routes (you can imagine an app with many different object types would get confusing like this)